### ISSUE TITLE
Security-issue: password hash and salt of provider exposed to customer

### ISSUE THREAD

**Issue by hermannschwaerzlerUIBK** [alextselegidis/easyappointments#208]
In the mail a customer receives after making an appointment there is a link to edit or cancel it.
When a customer follows that link the opened page contains an object named `GlobalVariables` in its DOM.

In this page this object looks like this (it contains much more data - I am showing the relevant one):
```
{
   "providerData": {
      "settings": { 
         "password": "e6a76969876555b824e17b6765430fbc5fabcded285e6a786da79515ec95c81e",
         "salt": "864b12345d38569c3271cba0456789e5a3382d0824af5eb5817867c7f78d4fc5",
         ...
      },
      ...
   },
   ...
}
```

I don't think this is a big problem as those are sha256 hashes. But still I think it's better to not expose them...

**Comment by alextselegidis** [comment]
Thanks for reporting, I'll correct this asap.

**Comment by alextselegidis** [comment]
Fixed with v1.2.0 Beta
